staging/src/kubevirt.io/client-go/api/v1/schema.go

```diff
@@ -428,6 +434,14 @@ type GPU struct {
 	DeviceName string `json:"deviceName"`
 }
 
+//
+// +k8s:openapi-gen=true
+type HostDevice struct {
+	// DeviceName is the resource name of the host device exposed by a device plugin
+	Name       string `json:"name"`
+	DeviceName string `json:"deviceName"`
```

Comment:
DeviceName comment needs to go above that struct.

Also, for `Name`, does that primarily exist simply to allow multiple entries of the same DeviceName to exist?

Reply:
> Also, for `Name`, does that primarily exist simply to allow multiple entries of the same DeviceName to exist?

Yes, exactly.

Reply:
> DeviceName comment needs to go above that struct.

I'll update.